### PR TITLE
OSDOCS-2228: Updated list of supported regions to match the updated list in OSD.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
JIRA: https://issues.redhat.com/browse/OSDOCS-2228

Preview: https://deploy-preview-33069--osdocs.netlify.app/openshift-rosa/latest/rosa_policy/rosa-service-definition#rosa-sdpolicy-regions-az_rosa-service-definition

Peer reviewer - No milestones/labels needed. Please merge to branch/ dedicated. Thanks.

### CODE REVIEW

File: modules/rosa-sdpolicy-account-management.adoc [hunk 1]
```diff
@@ -99,10 +101,11 @@ The following AWS regions are supported by Red Hat OpenShift 4 and are supported
 - ca-central-1 (Central)
```
Comment (wgordon17) [inline modules/rosa-sdpolicy-account-management.adoc:101]: This should probably be "Central Canada", I'm going to update the OSD Service Definition now

Comment (jonquilwilliams) [reply modules/rosa-sdpolicy-account-management.adoc]: You got it.

**Comment by lpettyjo** [comment]
LGTM

**Comment by mburke5678** [comment]
LGTM

**Pull request merged**